ui/lib/utils/sentryHelpers.ts

```diff
@@ -43,6 +43,8 @@ export const reportError: typeof Sentry.captureException = (...args) => {
 }
 
 export function initSentryRoutingInstrument() {
+  const firstMountTransaction = Sentry.startTransaction({ name: 'first mount' })
```

Comment:
Move here to fix the runtime crash, `firstMountTransaction` is undefined when it is initialized in the outside.